OWNERS.md

```diff
@@ -58,8 +57,6 @@ routing PRs, questions, etc. to the right place.
 This section lists a few people that are not maintainers but typically help out with subject
 matter expert reviews. Feel free to loop them in as needed.
 
-* Piotr Sikora ([PiotrSikora](https://github.com/PiotrSikora)) (piotrsikora@google.com)
-  * TLS, BoringSSL, and core networking (listeners, connections, etc.).
```

Comment:
Keep your name here and change it to WASM? :)

Reply:
I could, but this whole section predates `CODEOWNERS` and it's quite outdated (e.g. listing Shriram), so perhaps it would be better to completely remove it and assign the code ownership to still active friends of Envoy?